crates/wasmparser/src/validator.rs

```diff
@@ -346,7 +346,7 @@ pub enum ValidPayload<'a> {
     /// This result indicates that the specified parser should be used instead
     /// of the currently-used parser until this returned one ends.
     Parser(Parser),
-    /// A function was found to be validate.
+    /// A function was found to be valid.
```

Comment:
Oh "validate" here should have been "validated", the function isn't valid yet in this variant and it still needs to be validated.

Reply:
Fixed and push -f'd!